### DISCUSSION TITLE
Adding Supplier

### DISCUSSION THREAD

**Discussion by LarryJoy** [inventree/InvenTree#8195] [Q&A] [answered]
Okay community. I am attempting, for the first time, to add suppliers. I got to the add supplier form, put in the information, and get "Form errors exist" when I submitted. There is no indication as to what information is needed.--Help me to understand the problem and what to do. Thank you.

**Comment by SchrodingersGat** [comment]
Can you provide a screenshot of the form please (it is always a good starting point for these discussions).

**Reply by LarryJoy** [reply]
Yes, I will provide a screenshot if I can figure out how to do it.

**Comment by LarryJoy** [comment]
![Screenshot from 2024-09-26 09-57-43](https://github.com/user-attachments/assets/f7bba41a-1470-45a4-a905-1b4da1c16022)
file:///home/testuser/Pictures/Screenshots/Screenshot%20from%202024-09-26%2009-57-43.png
Here it is.

**Reply by wolflu05** [reply, marked as answer]
I suspect that you have to use a valid url in form of https://www.mouser.com

**Comment by LarryJoy** [comment]
That worked. Thank you.

**Comment by LarryJoy** [comment]
Yes, I edited the information using a complete URL for Mouser, https://www.mouser.com and that cleared the error. Thanks again.
